python/ola/MACAddress.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, mac_address):
 
     Args:
       mac_address: The byte array representation of the MAC Address, e.g.
-      bytearray([0x01, 0x23, 0x45, 0x67, 0x89 0xab]).
+      bytearray([0x01, 0x23, 0x45, 0x67, 0x89, 0xab]).
```

Comment:
Good spot!

Reply:
I don't suppose there's any equivalent to Doxygen code samples where we can validate that this example code compiles in Python land?